content/en/docs/concepts/configuration/assign-pod-node.md

```diff
@@ -27,9 +27,35 @@ repo here](https://github.com/kubernetes/website/tree/{{< param "docsbranch" >}}
 
 {{% capture body %}}
 
+## nodeName
+
+`nodeName` is the simplest node selection constraint.
+`nodeName` is a field of PodSpec.  If it is non-empty, the scheduler simply schedules this pod onto that node,
```

Comment:
This is not exactly correct. When `nodeName` is non-empty, the scheduler ignores the pod. It is the corresponding kubelet that picks up the pod and tries to run it if it fits.

Reply:
That part of sentence was taken directly from the API docs, https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.13/#podspec-v1-core , but I'll change it.